utils/build-script-impl

```diff
@@ -1468,7 +1468,12 @@ function copy_embedded_compiler_rt_builtins_from_darwin_host_toolchain() {
                 DEST_LIB_PATH="${DEST_BUILTINS_DIR}/${LIB_NAME}"
                 if [[ ! -f "${DEST_LIB_PATH}" ]]; then
                     if [[ -f "${HOST_LIB_PATH}" ]]; then
-                        call cp "${HOST_LIB_PATH}" "${DEST_LIB_PATH}"
+                        if [[ "$OS" == "tvos" ]]; then
+                            # This is to avoid strip failures when generating a toolchain
+                            call lipo -remove i386 "${HOST_LIB_PATH}" -output "${DEST_LIB_PATH}" || call cp "${HOST_LIB_PATH}" "${DEST_LIB_PATH}"
```

Comment:
This line does not behave as expected when the source file does not have the i386 slice to begin with (as it's the case with Xcode 13.0 beta 1)

In addition, this would print error messages in the log for a scenario that is handled by the code.

Reply:
This is addressed in 376b6f20ebc